src/core/Window.cpp

```diff
@@ -99,6 +99,8 @@ void Window::Initialize(const std::vector<std::string>& otrFiles, const std::uno
     InitializeControlDeck();
     InitializeCrashHandler();
 
+    CVarLoad();
```

Comment:
This was moved from the ImGui `Init()` because it made absolutely no sense there.